rrfs_ci: Updates from develop

christinaholtNOAA opened the pull request:
## DESCRIPTION OF CHANGES: 
Update to the top of ufs-community develop.

There are some wonky looking things in this PR because I squashed the last merge accidentally. This PR should not be squashed.


## TESTS CONDUCTED: 
None

## DEPENDENCIES:
[PR #165](https://github.com/NOAA-GSL/ufs-srweather-app/pull/165) in NOAA-GSL/ufs-srweather-app

JeffBeck-NOAA committed:
Add missing user-defined stochastic physics options; fix stochastic physics seed generation script (#704)

## DESCRIPTION OF CHANGES: 
Add missing user-defined options for tendency-based stochastic physics and fix the ensemble-based seed generation script to work regardless of whether stochastic physics is turned on or not.

## TESTS CONDUCTED: 
Tested on Hera using the following WE2E configurations with and without stochastic physics:

config.grid_RRFS_CONUS_3km_ics_HRRR_lbcs_RAP_suite_HRRR.sh
config.community_ensemble_2mems.sh

## ISSUE (optional): 
[Issue #702](https://github.com/ufs-community/regional_workflow/issues/702)

## CONTRIBUTORS (optional): 
Thanks to @mkavulich and @chan-hoo for finding this problem.

JeffBeck-NOAA committed:
Add namelist option for netCDF4 when running with the 3-km NA domain; update NAM HPSS settings and WE2E tests (#707)

* Change to netcdf4 when using the NA 3-km domain

* Update HPSS paths for NAM data

* Update NAM HPSS locations and dates for WE2E tests.

* Remove lines from merge.

mark-a-potts committed:
Tweaks to allow compiler and build_env_fn to be specified in the run_WE2E_test.sh script (#711)

michelleharrold committed:
Changed 20200304 to 20200303 in ush/mrms_pull_topofhour.py (#712)

christinaholtNOAA committed:
Remove unused rocoto directory in ush (#720)

chan-hoo committed:
Fix bug for nco we2e tests on Orion; re-organize we2e input data and nco we2e tests (#713)

* Update machine script for orion

* Update machine script for wcoss_dell_p3

* Update we2e run script for wcoss and orion

* Reorganize nco we2e tests

* remove machine based logic

* Add symlink for nco inline post test

michelleharrold committed:
Added stand-alone verification scripts (feature/issue_683_standaloneVX) (#726)

* Grid-stat and point-stat run scripts.

* Stand-alone scripts for verification.

* Added comments to gridvx scripts.

* Added qsub_job.sh and added comments to provide context on running Vx.

chan-hoo committed:
remove machine base logic (#727)

chan-hoo committed:
Allow user-defined file names for input template files (#717)

* Allow multiple template names

* parameterize file_TMPL_FN and add a we2e test

* Increase maxtries_task for make_grid/orog/sfc_climo

* Modify file name and description

JeffBeck-NOAA committed:
Changes to RRFS 3- and 13-km domains, setup.sh script bug fixes, make_ics task modification, and tweaks to stochastic physics namelist settings (#721)

* Modify RRFS North America 3- and 13-km domain configuration and WE2E test.

* Change sotyp_from_climo to "true" based on operational RAP grib2 files.

* Update for changes to stochastic physics namelist options.

* Check for DO_ENSEMBLE="TRUE" when running ensemble verification and turn of VX when running in NCO mode.

* Revert to 3-km domain.

* Remove commented-out GFDL grid for the RRFS_NA_13km domain

* Add RRFS_NA_13km WE2E test

* Changes to comments.

christinaholtNOAA committed:
Adding 25 km tests to Jet/Hera suites. (#718)

gsketefian committed:
Add a small 3km predefined grid over Indianapolis for testing (#725)

* Add 3km grid over Indianapolis.  This is about 600km x 600km in extent (200 x 200 grid points).  It is intended for use in the WE2E tests.

* Edit comments.

christinaholtNOAA committed:
Use Python tool for get_extrnl_mdl_file tasks (#681)

These changes hook in the Python-based data ingest tool, replacing the previous scripts that handled this work as part of the get_extrn_mdl_file task. No attempt was made in this PR to replace the NOMADS fetching script with the Python utility, but the NOMADS data location has been added to the data_locations.yml file.

The functionality to write the data summary file has also been added to the Python tool to match the capabilities of the existing workflow tools.

BenjaminBlake-NOAA committed:
Increase size of RRFS CONUS grid (#724)

Co-authored-by: Benjamin.Blake EMC <Benjamin.Blake@v71a1.ncep.noaa.gov>
Co-authored-by: Benjamin.Blake EMC <Benjamin.Blake@v72a1.ncep.noaa.gov>
Co-authored-by: Benjamin.Blake EMC <Benjamin.Blake@v71a3.ncep.noaa.gov>
Co-authored-by: chan-hoo <chan-hoo.jeon@noaa.gov>

willmayfield committed:
add include-style quality mark options in metplus confs (#738)

mark-a-potts committed:
Add Gaea as a supported platform for the regional_workflow (#734)

* Updates to port regional workflow to gaea

* Temp change with -v as batch option

* new fixes for gaea/slurm

* Updated time for make lbcs

* added TEST data directory path

* Update gaea.sh

* fixes for PR

gsketefian committed:
Add more parameters to CSV file containing WE2E test info (#740)

## DESCRIPTION OF CHANGES: 
The script/function `get_WE2Etest_names_subdirs_descs.sh` (which is called from `run_WE2E_tests.sh` if needed) creates a CSV (Comma-Separated Value) file named `WE2E_test_info.csv` that contains information about the WE2E tests.  Currently, this CSV file contains only 3 columns: the test name, the names of any alternate names for the test, and the test description.  In order to have a more complete summary of the WE2E tests, this PR modifies `get_WE2Etest_names_subdirs_descs.sh` so that additional information is included in the CSV file.  This additional information consists of the values of the following experiment variables for each test:
```
PREDEF_GRID_NAME
CCPP_PHYS_SUITE
EXTRN_MDL_NAME_ICS
EXTRN_MDL_NAME_LBCS
DATE_FIRST_CYCL
DATE_LAST_CYCL
CYCL_HRS
INCR_CYCL_FREQ
FCST_LEN_HRS
LBC_SPEC_INTVL_HRS
NUM_ENS_MEMBERS
```
In addition, the script uses this information to calculate the number of times each test calls the forecast model (e.g. if the test uses 3 different cycle dates, then the forecast model will be called 3 times; if it is an ensemble test for a single cycle, the test will call the forecast model as many times as the number of ensemble members).  

## TESTS CONDUCTED: 
The script `run_WE2E_tests.sh` was called that in turn calls `get_WE2Etest_names_subdirs_descs.sh`.  This created a new CSV file that contained the new fields (columns).  The CSV file was imported into Google Sheets (using "|" as the field/column separator) and looked correct.

## DOCUMENTATION:
The documentation is for the most part already within the `get_WE2Etest_names_subdirs_descs.sh`.  This PR slightly modifies that documentation to update it.

chan-hoo committed:
Update directory structure of NCO mode (#743)

* update vertical structure of NCO mode

* update sample script for nco

* Fix typo on write component of new RRFS CONUS

natalie-perlin committed:
Default CCPP physics option is FV3_GFS_v16 (#746)

* Updated the default CCPP physics option to FV3_GFS_v16

* Updated the default CCPP physics option to FV3_GFS_v16 in config_defaults.sh

Co-authored-by: Natalie Perlin <Natalie@Natalies-MacBook-Air.local>

danielabdi-noaa committed:
Adds an alternative python workflow generation path (#698)

* Workflow in python starting to work.

* Use new python_utils package structure.

* Some bug fixes.

* Use uppercase TRUE/FALSE in var_dfns

* Use config.sh by default.

* Minor bug fixes.

* Remove config.yaml

* Update to the latest develop

* Remove quotes from numbers in predef grid.

* Minor bug fix.

* Move validity checker to the bottom of setup

* Add more unit tests.

* Update with python_utils changes.

* Update to latest develop additions (Need to re-run regression test)

* Use set_namelist and fill_jinja_template as python functions.

* Replace sed regex searches with python re.

* Use python realpath.

* Construct settings as dictionary before passing to fill_jinja and set_namelist

* Use yaml for setting predefined grid parameters.

* Use xml parser for ccpp phys suite definition file.

* Remove more run_command calls.

* Simplify some func argument processing.

* Move different config format parsers to same file.

* Use os.path.join for the sake of macosx

* Remove remaining func argument processing via os.environ.

* Minor bug fix in set_extrn_mdl_params.sh

* Add suite defn in test_data.

* Minor fixes on unittest on jet.

* Simplify boolean condition checks.

* Include old in renaming of old directories

* Fix conflicting yaml !join tag for paths and strings.

* Bug fix with setting sfcperst dict.

* Imitate "readlink -m" with os.path.realpath instead of os.readlink

* Don't use /tmp as that is shared by multiple users.

* Bug fix with cron line, maintain quotes around TRUE/FALSE.

* Update to latest develop (untested)

* Bug fix with existing cron line and quotes.

* Bug fix with case-sensitive MACHINE name, and empty EXPT_DIR.

* Update to latest develop

* More updates.

* Bug fix thanks to @willmayfield! Check both starting/ending
characters are brackets for shell variable to be considered an array.

* Make empty EXPT_BASEDIR workable.

* Update to latest develop

* Update in predef grid.

* Check f90nml as well.

Co-authored-by: Daniel Abdi <dabdi@Orion-login-2.HPC.MsState.Edu>

chan-hoo committed:
Fix typo and crontab issue on wcoss dell in workflow python scripts (#750)

* Fix typo and failure on wcoss

* fix new line issue on wcoss dell

* remove capture_output

* Get USER from environment

Co-authored-by: Daniel Abdi <daniel.abdi@noaa.gov>

EdwardSnyder-NOAA committed:
Add new WE2E configs (#748)

## DESCRIPTION OF CHANGES: 
Added two new WE2E config files for the Sub-CONUS Indianapolis domain to support the upcoming SRW release. 

In addition, modified the external data used in the `config.specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS.sh` to match more common datasets used in the WE2E testing process. 

## TESTS CONDUCTED: 
Successfully ran the new WE2E tests (`config.SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_HRRR.sh`, `config.SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta.sh`) and `config.specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS.sh` on NOAA Parallel Works AWS instance.

## DEPENDENCIES:
None.

## DOCUMENTATION:
No documentation changes are required.

ywangwof committed:
Added a fixed WoF grid and the python tool to determine the write component parameters (#733)

* Added a fixed WoF grid and the python tool to determine the write component parameters

* Update set_predef_grid_params.sh

* Renamed file as recommended and removed unused lines

* Modified comment

Co-authored-by: JeffBeck-NOAA <55201531+JeffBeck-NOAA@users.noreply.github.com>
Co-authored-by: WYH@MBP <yunheng.wang@4374-ywang.winstorm.nssl>

chan-hoo committed:
Replace env with modulefiles in scripts (#752)

* change env to mod

* update we2e script

gsketefian committed:
WE2E script improvements for usability (#745)

## DESCRIPTION OF CHANGES: 
* Modifications to `run_WE2E_tests.sh`:
  * Add examples to help/usage statement
* Modifications to `check_expts_status.sh`:
  * Add arguments list that can be processed by `process_args`
  * Add new optional arguments:  `num_log_lines`, `verbose`
  * Include a help/usage message

## TESTS CONDUCTED:
* Ran `run_WE2E_tests.sh --help` from the command line and got the expected help message.
* Ran `check_expts_status.sh --help` from the command line and got the expected help message.
* Used `run_WE2E_tests.sh` to run a set of 2 WE2E tests -- works as expected.
* Used `check_expts_status` to check on the status of the 2 tests run above and got the expected status message.
 
## DEPENDENCIES:
PR #[241](https://github.com/ufs-community/ufs-srweather-app/pull/241)

## DOCUMENTATION:
A lot of this PR is documentation in the scripts.  There is an accompanying documentation PR #[241](https://github.com/ufs-community/ufs-srweather-app/pull/241) into ufs-srweather-app.

mkavulich committed:
Standardize static data across Tier-1 platforms; fix and improve IC and LBC data retrieval (#744)

christinaholtNOAA committed:
Merge remote-tracking branch 'ufs/develop' into update_rrfs_ci

gsketefian committed:
Bug fixes (grid size + suppress screen output from module load) (#756)

## DESCRIPTION OF CHANGES: 
1) Adjust y-direction size of write-component grid of `SUBCONUS_Ind_3km` predefined grid from 195 to 197 (this was just an oversight in PR #725 ).
2) Redirect output of module load in launch script (`launch_FV3LAM_wflow.sh`) to `/dev/null` to avoid unwanted screen output (which was introduced in PR #[238](https://github.com/ufs-community/ufs-srweather-app/pull/238) in ufs-srweather-app and is about how to load the `regional_workflow` environment and is not relevant in this context).

## TESTS CONDUCTED: 
1) Plotted the `SUBCONUS_Ind_3km` grid to ensure it has correct size (it does).
2) Manually ran `launch_FV3LAM_wflow.sh` from the command line to verify that screen output is suppressed (it is).

JeffBeck-NOAA committed:
Update default SPP ISEED array in config_defaults.sh to use unique values (#759)

* Modify RRFS North America 3- and 13-km domain configuration and WE2E test.

* Modify default ISEED values for SPP

* Fix grid in WE2E test

chan-hoo committed:
Update workflow python scripts (#760)

* update python scripts

chan-hoo committed:
Change output file name of run_post to meet NCO standards (#758)

* change output file name

* change variable name

* update python script

* remove duplicates

* add a check for empty variables

* move variable to common area

* clean up unnecessary comments

* update scripts

* remove duplicate

* update python scripts

* fix user-staged dir path issue in python script

chan-hoo committed:
Add POST_OUTPUT_DOMAIN_NAME to WE2E tests for new grids (#763)

* Add new var to we2e tests for new grids

* rename we2e tests for custom grid

* remove unnecessary $

gsketefian committed:
Modifications to `CODEOWNERS` file (#757)

* Add @gspetro-NOAA, @natalie-perlin, and @EdwardSnyder-NOAA to CODEOWNERS so they are notified of all PRs and can review them.

* Remove duplicates in CODEOWNERS; remove users who will no longer be working with the repo.

christinaholtNOAA committed:
Adding a python utility for summarizing compute. (#769)

Adds a utility that summarizes Rocoto database computational usage information.

danielabdi-noaa committed:
Add github actions for python unittests. (#747)

* Add github actions for python unittests.

* Include all python script in ush

* Skip defining QUILTING params when it is set to False

* Update py_workflow

* Update unittest for set_extrn_mdl_params.

* Updates from develop.

Co-authored-by: Daniel Shawul <dshawul@yahoo.com>

chan-hoo committed:
Update sample script for NCO mode (#771)

* update config.nco.sh

* Add comment

mark-a-potts committed:
Feature/noaacloud (#767)

* updates for noaacloud

* working version

* fixes for noaacloud

* added extra modules for post

mark-a-potts committed:
removed cheyenne-specific crontab editing section (#773)

christinaholtNOAA committed:
Pin down hera miniconda3 module file version. (#770)

Pin down the version of miniconda3 on Hera, and do not append to the module path.

EdwardSnyder-NOAA committed:
update staged data dir (#774)

christinaholtNOAA committed:
Merge remote-tracking branch 'ufs/develop' into update_rrfs_ci

EdwardSnyder-NOAA committed:
update IC/LBC file paths to match new IC/LBC file structure (#766)

* update IC/LBC file paths to match new IC/LBC file structure

* change forecast length

* Update config.community.sh

These changes reflects the SRW team's preference for a 12-hours forecast. The gfs.t18z.pgrb2.0p25.f012 file will need to be added to all Level 1 systems.

Co-authored-by: Gillian Petro <96886803+gspetro-NOAA@users.noreply.github.com>

natalie-perlin committed:
add a machine file for MacOS (#777)

Co-authored-by: Natalie Perlin <perlin.natalie@gmail.com>

gsketefian committed:
Add columns for relative test and time step to CSV file containing WE2E test info (#776)

## DESCRIPTION OF CHANGES: 
This PR modifies the script `get_WE2Etest_names_subdirs_descs.sh` that creates the CSV file containing test information the columns for relative cost and time step (`DT_ATMOS`).  This allows users to have a better idea of what the cost of each test is (the relative cost is defined such that 1 unit represents the cost of running a single 6-hour forecast on the `RRFS_CONUS_25km` grid).  

Other related changes:
* Add the flag `generate_csv_file` to the `run_WE2E_tests.sh` script to allow users to skip the CSV file generation step (since it can take a while).
* Use arguments for the function `set_predef_grid_params()` so that it can be called not just from the experiment generation scripts but also others (in this case from `get_WE2Etest_names_subdirs_descs.sh`).  Necessary for this PR.
* Move some constants (`NH0`, `NH3`, `NH4`) from `config_defaults.sh` to `constants.sh` to make it easier to access them.
* Place the contents of `constants.sh` at the start of each experiment's `var_defns.sh` so that these constants are available to the j-jobs and ex-scripts when `var_defns.sh` is sourced.
* Use variable `valid_vals_BOOLEAN` in many more places (which allows many other `valid_vals_...` variables in `valid_param_vals.sh` and `setup.sh` to be removed).
* Several other bug fixes.

## TESTS CONDUCTED: 
The following tests (with results) were conducted on Hera.  There were only three (previously existing) failures.
```
* grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16:               SUCCESS
* grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp:           SUCCESS
* grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional:  SUCCESS
* grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2:                 SUCCESS
* grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16:                   SUCCESS
* grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR:                         SUCCESS
* grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_2017_gfdlmp:           SUCCESS
* grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2:                 SUCCESS
* grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v16:                   FAILURE
* grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_HRRR:                            SUCCESS
* grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_RRFS_v1beta:                     SUCCESS
* grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16:            SUCCESS
* grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR:                   SUCCESS
* grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_RRFS_v1beta:            SUCCESS
* grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR:                    SUCCESS
* grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1alpha:            SUCCESS
* grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta:             SUCCESS
* grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16:                  SUCCESS
* grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_HRRR:                          SUCCESS
* grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta:                   SUCCESS
* GST_release_public_v1:                                                       SUCCESS
* MET_ensemble_verification:                                                   SUCCESS
* MET_verification:                                                            SUCCESS
* community_ensemble_008mems:                                                  SUCCESS
* community_ensemble_2mems:                                                    SUCCESS
* custom_ESGgrid:                                                              SUCCESS
* custom_GFDLgrid:                                                             SUCCESS
* custom_GFDLgrid__GFDLgrid_USE_GFDLgrid_RES_IN_FILENAMES_eq_FALSE:            SUCCESS
* custom_GFDLgrid__GFDLgrid_USE_GFDLgrid_RES_IN_FILENAMES_eq_TRUE:             SUCCESS
* deactivate_tasks:                                                            SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2019061200:                   SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2019101818:                   SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2020022518:                   SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2020022600:                   SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2021010100:                   SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio:                             SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2019061200:                  SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2019101818:                  SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2020022518:                  SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2020022600:                  SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2021010100:                  SUCCESS
* get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_netcdf_2021062000:                  SUCCESS
* get_from_HPSS_ics_GSMGFS_lbcs_GSMGFS:                                        SUCCESS
* get_from_HPSS_ics_HRRR_lbcs_RAP:                                             SUCCESS
* get_from_HPSS_ics_RAP_lbcs_RAP:                                              SUCCESS
* get_from_NOMADS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio:                           SUCCESS
* inline_post:                                                                 SUCCESS
* nco_ensemble:                                                                SUCCESS
* nco_inline_post:                                                             SUCCESS
* pregen_grid_orog_sfc_climo:                                                  SUCCESS
* specify_DOT_OR_USCORE:                                                       SUCCESS
* specify_DT_ATMOS_LAYOUT_XY_BLOCKSIZE:                                        SUCCESS
* specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS:                                       SUCCESS
* specify_RESTART_INTERVAL:                                                    SUCCESS
* specify_template_filenames:                                                  SUCCESS
* subhourly_post:                                                              FAILURE
* subhourly_post_ensemble_2mems:                                               FAILURE
```

## DOCUMENTATION:
Documentation is updated via PR #[278](https://github.com/ufs-community/ufs-srweather-app/pull/278) in the ``ufs-srweather-app`` repo.

gsketefian committed:
Bug fix for calculation of number of time steps for a given WE2E test. (#782)

## DESCRIPTION OF CHANGES: 
This PR fixes a bug in the way the number of time steps (`num_time_steps` in `get_WE2Etest_names_subdirs_descs.sh`) is calculated for each WE2E test when creating the CSV file containing information about the available WE2E tests.

## TESTS CONDUCTED: 
Reran the `run_WE2E_tests.sh` script (which in turn calls `get_WE2Etest_names_subdirs_descs.sh`), and a CSV file containing the adjusted numbers was successfully created.

## DOCUMENTATION:
None needed.

gsketefian committed:
Bugfix:  Pass domain name to python plotting scripts (#783)

## DESCRIPTION OF CHANGES: 
PR #[763](https://github.com/ufs-community/regional_workflow/pull/763) introduced the experiment variable `POST_OUTPUT_DOMAIN_NAME` and used it to modify the names of the UPP output files.  This PR adds `POST_OUTPUT_DOMAIN_NAME` as an argument to the python plotting scripts to enable those scripts to form the proper UPP output file names to read in.  It also modifies the batch scripts (`qsub_job.sh`, `qsub_job_diff.sh`, `sq_job.sh`, and `sq_job_diff.sh`) to add the new argument to the calls to the python plotting scripts that are called.

## TESTS CONDUCTED: 
Both python plotting scripts `plot_allvars.py` and `plot_allvars_diff.py` were run on Hera on preexisting UPP output files and completed successfully.  The batch scripts `sq_job.sh` and `sq_job_diff.sh` were also run successfully on Hera (`qsub_job.sh` and `qsub_job_diff.sh` were not run on e.g. Cheyenne).

## DEPENDENCIES:
PR # in `ufs-srweather-app`.

## DOCUMENTATION:
Modifications to the documentation are in PR #.

## ISSUE: 
Partially resolves issue #781 (still need to introduce tests in `regional_workflow` that run the plotting scripts).

## CONTRIBUTORS: 
@mark-a-potts pointed out the problem and first made the necessary fixes in the `NOAA-EPIC` fork of `regional_workflow`; @gsketefian introduced those changes to this PR and made edits to the comments and documentation.

chan-hoo committed:
Update scripts to work with the latest hashes of UFS_UTILS and UPP (#775)

* update input namelist of chgres_cube

* update diag_table templates

* update scripts

* back to original

* specify miniconda version on Jet

EdwardSnyder-NOAA committed:
adding new e2e tests (#792)

chan-hoo committed:
[develop] Update scripts to run with the latest hash of ufs weather model (#796)

* update new variables introduced in uwm

* update python script

* update template for HRRR suite

willmayfield committed:
[develop] Update MET/METplus versions and configuration files (#779)

* remove unused module loads for MET

* update met version in machine files

* changes to pointstat/pb2nc bufr mapping

* add NA to retain bufr variables without qc markers

* updated bufr variable list

* change bufr var list

* Change versions to MET 10.1.1 and METplus 4.1.1

gspetro-NOAA committed:
add a linux.sh machine file (a template) (#798) (#803)

Co-authored-by: Natalie Perlin <68030316+natalie-perlin@users.noreply.github.com>

gsketefian committed:
[develop] Rename variables related to GFDLgrid-type grids (#787)

## DESCRIPTION OF CHANGES: 
For clarity, rename variables related to GFDLgrid-type grids:
1) Rename `GFDLgrid_RES` to `GFDLgrid_NUM_CELLS`.
2) Rename `GFDLgrid_USE_GFDLgrid_RES_IN_FILENAMES` to `GFDLgrid_USE_NUM_CELLS_IN_FILENAMES`.
3) Rename WE2E tests on GFDLgrid-type custom grids according to variable renaming above.

Related improvements:
* In `link_fix.sh`, remove creation of unnecessary symlink when using a GFDLgrid-type grid with `GFDLgrid_USE_NUM_CELLS_IN_FILENAMES` is set to `"FALSE"`.
* In WE2E test configuration files that use custom grids, move the definition of the variable `POST_OUTPUT_DOMAIN_NAME` to the beginning of the custom grid definition because it is really the name of the custom native grid, and it's most appropriate to place that at the beginning of the grid definition section.
* Reduce `WTIME_MAKE_OROG` from 1 hour to 20 minutes since on Hera, 1 hour causes the task to wait quite a while in the queue.  This used to be 20 minutes but was changed to 1 hour because 20 minutes was not sufficient to complete the task on `WCOSS_DELL_P3`.  Thus, reset it to 1 hour only in the machine file for `WCOSS_DELL_P3`.
* Minor fixes to comments in python plotting scripts.
* Edits to other comments.

## TESTS CONDUCTED: 
Ran the four WE2E tests that use custom grids (listed below) on Hera.  All were successful.
* `custom_ESGgrid`
* `custom_GFDLgrid`
* `custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE` (previously `custom_GFDLgrid__GFDLgrid_USE_GFDLgrid_RES_IN_FILENAMES_eq_FALSE`)
* `custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_TRUE` (previously `custom_GFDLgrid__GFDLgrid_USE_GFDLgrid_RES_IN_FILENAMES_eq_TRUE`)

## DEPENDENCIES:
PR #[310](https://github.com/ufs-community/ufs-srweather-app/pull/310) in `ufs-srweather-app`.

## DOCUMENTATION:
Updates to documentation are in PR #[310](https://github.com/ufs-community/ufs-srweather-app/pull/310) in `ufs-srweather-app`.

venitahagerty commented:
Machine: hera
Compiler: intel
Job: WE
Repo location: /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app
  /apps/cmake/3.20.1/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/chgres_cube does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/emcsfc_ice_blend does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/emcsfc_snow2mdl does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/filter_topo does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/fregrid does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/fvcom_to_FV3 does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/global_cycle does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/global_equiv_resol does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/inland does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/lakefrac does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/make_hgrid does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/make_solo_mosaic does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/orog does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/orog_gsl does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/regional_esg_grid does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/sfc_climo_gen does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/shave does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/vcoord_gen does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/adjust_soiltq.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/check_imssnow_fv3lam.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/fv3lam_nonvarcldana.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/gen_annual_maxmin_GVF.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/gen_cs.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/gen_ensmean_recenter.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/lakesurgery.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_imssnow_fv3lam.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_larccld.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_Lightning.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_metarcld.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_NSSL_mosaic.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/process_updatesst.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/ref2tten.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/update_bc.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/update_GVF.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/update_ice.exe does NOT exist'
+ echo 'FAIL: intel executable file /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220629145011/ufs-srweather-app/test/../bin_intel/bin/use_raphrrr_sfc.exe does NOT exist'
+ echo 'BUILD(S) FAILED'
+ msg=FAIL
+ echo FAIL
Build failed
If test failed, please make changes and add the following label back:
ci-hera-intel-WE

venitahagerty commented:
Found this message in build.out on Hera:
CMake Error at /apps/cmake/3.20.1/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find wgrib2 (missing: WGRIB2_INCLUDES WGRIB2_EXE) (Required is at
  least version "2.0.8")
Call Stack (most recent call first):
  /apps/cmake/3.20.1/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/Findwgrib2.cmake:38 (find_package_handle_standard_args)
  CMakeLists.txt:73 (find_package)

venitahagerty commented:
Machine: jet
Compiler: intel
Job: WE
Repo location: /lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app
  /misc/apps/cmake/3.20.1/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/chgres_cube does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/emcsfc_ice_blend does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/emcsfc_snow2mdl does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/filter_topo does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/fregrid does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/fvcom_to_FV3 does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/global_cycle does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/global_equiv_resol does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/inland does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/lakefrac does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/make_hgrid does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/make_solo_mosaic does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/orog does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/orog_gsl does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/regional_esg_grid does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/sfc_climo_gen does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/shave does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/vcoord_gen does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/adjust_soiltq.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/check_imssnow_fv3lam.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/fv3lam_nonvarcldana.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/gen_annual_maxmin_GVF.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/gen_cs.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/gen_ensmean_recenter.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/lakesurgery.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_imssnow_fv3lam.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_larccld.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_Lightning.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_metarcld.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_NSSL_mosaic.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/process_updatesst.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/ref2tten.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/update_bc.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/update_GVF.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/update_ice.exe does NOT exist'
+ echo 'FAIL: intel executable file /mnt/lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220629162014/ufs-srweather-app/test/../bin_intel/bin/use_raphrrr_sfc.exe does NOT exist'
+ echo 'BUILD(S) FAILED'
+ msg=FAIL
+ echo FAIL
Build failed
If test failed, please make changes and add the following label back:
ci-jet-intel-WE

christinaholtNOAA committed:
Remove un-readable path to systemdir on Hera. (#806)

This is a bug fix for running the WE2E tests on Hera. The non-existence of this path is causing failures after the path has become unreadable.

EdwardSnyder-NOAA committed:
[develop] Add MET/METplus to Jet (#799)

* add metplus paths

* add run_vx.local for jet

Co-authored-by: Edward Snyder <Edward.Snyder@noaa.com>

robgpita commented:
The build test passed on PCluster. @venitahagerty, I wonder why it couldn't find `wgrib2` on hera.

christinaholtNOAA committed:
Merge remote-tracking branch 'ufs/develop' into update_rrfs_ci

venitahagerty commented:
Machine: hera
Compiler: intel
Job: WE
Repo location: /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/982670352/20220711212015/ufs-srweather-app
Build was Successful
Rocoto jobs started
Long term tracking will be done on 10 experiments
If test failed, please make changes and add the following label back:
ci-hera-intel-WE
Experiment Failed on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
2022-07-11 21:54:06 +0000 :: hfe12 :: Task make_lbcs, jobid=33647577, in state DEAD (FAILED), ran for 7.0 seconds, exit status=256, try=2 (of 2)
Experiment Failed on hera: nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
2022-07-11 21:54:08 +0000 :: hfe06 :: Task make_lbcs, jobid=33647574, in state DEAD (FAILED), ran for 8.0 seconds, exit status=256, try=2 (of 2)
Experiment Failed on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1alpha
2022-07-11 21:58:07 +0000 :: hfe02 :: Task make_lbcs, jobid=33647690, in state DEAD (FAILED), ran for 8.0 seconds, exit status=256, try=2 (of 2)
Experiment Failed on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
2022-07-11 21:56:10 +0000 :: hfe09 :: Task make_lbcs, jobid=33647634, in state DEAD (FAILED), ran for 6.0 seconds, exit status=256, try=2 (of 2)
Experiment Failed on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
2022-07-11 21:58:07 +0000 :: hfe05 :: Task make_lbcs, jobid=33647688, in state DEAD (FAILED), ran for 8.0 seconds, exit status=256, try=2 (of 2)
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
Experiment Succeeded on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
Experiment Succeeded on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_RRFS_v1beta
All experiments completed

venitahagerty commented:
Machine: jet
Compiler: intel
Job: WE
Repo location: /lfs1/BMC/nrtrr/rrfs_ci/autoci/pr/982670352/20220711212018/ufs-srweather-app
Build was Successful
Rocoto jobs started
Long term tracking will be done on 10 experiments
If test failed, please make changes and add the following label back:
ci-jet-intel-WE
Experiment Failed on jet: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
2022-07-11 22:32:15 +0000 :: fe2 :: Task make_lbcs, jobid=6751065, in state DEAD (FAILED), ran for 156.0 seconds, exit status=256, try=1 (of 1)
Experiment Failed on jet: nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
2022-07-11 22:34:14 +0000 :: fe4 :: Task make_lbcs, jobid=6750766, in state DEAD (FAILED), ran for 477.0 seconds, exit status=256, try=1 (of 1)
Experiment Failed on jet: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
2022-07-11 22:28:17 +0000 :: fe4 :: Task make_lbcs, jobid=6750778, in state DEAD (FAILED), ran for 74.0 seconds, exit status=256, try=1 (of 1)
Experiment Failed on jet: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1alpha
2022-07-11 22:30:08 +0000 :: fe1 :: Task make_lbcs, jobid=6751070, in state DEAD (FAILED), ran for 92.0 seconds, exit status=256, try=1 (of 1)
Experiment Failed on jet: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
2022-07-11 22:34:11 +0000 :: fe4 :: Task make_lbcs, jobid=6751078, in state DEAD (FAILED), ran for 289.0 seconds, exit status=256, try=1 (of 1)
Experiment Succeeded on jet: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
Experiment Succeeded on jet: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
Experiment Succeeded on jet: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_RRFS_v1beta
Experiment Succeeded on jet: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
Experiment Succeeded on jet: grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2
All experiments completed

christinaholtNOAA commented:
This PR has gotten a bit convoluted after I squashed a previous merge from ufs/develop. Opening a new one to start fresh.
